src/analyze/analyze-security.c

```diff
@@ -1626,6 +1627,10 @@ static int assess(const struct security_info *info, Table *overview_table, Analy
                         return table_log_add_error(r);
         }
 
+        /* Return error when overall exposure level is over MEDIUM */
+        if (exposure > 50)
```

Comment:
to keep backward compat, maybe add a new ANALYZE_SECURITY_ERROR_ABOVE_MEDIUM or so enum, and do this only if set (and pass it when calling from verify_units)? That way the security verb does not change